cockatrice/src/messagelogwidget.cpp

```diff
@@ -364,13 +364,20 @@ void MessageLogWidget::logMoveCard(Player *player,
 
 void MessageLogWidget::logDrawCards(Player *player, int number)
 {
+    int deckSize = player->getZones().value("deck")->getCards().size();
```

Comment:
considering the player itself is the only one to ever emit the signal this connects to I think it'd make more sense to simply add it to the signal and this function as an extra parameter.